cves/CVE-2012-2775.yml

```diff
@@ -127,7 +130,12 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    Thilo Borggman originally authored the ALS decoder and his changes were
+    merged shortly after. There were a few errors and bug reports across the
+    lifetime of this vulnerability, but it was more or less stable. ALS is the
+    lossless audio format for MP4 and adding this library allowed the project
+    to decode MP4 videos which is a big feature.
```

Comment:
This sounds like the error was shortly after the original feature? Or was there a long time between the two?